r/R/download_annot_seurat_object.R

```diff
@@ -38,11 +41,19 @@ add_cellsets <- function(scdata, cellsets) {
     children <- cellset$children
     if (!length(children)) next()
 
-    # use key for name of column
-    key <- cellset$key
+    uuid_pattern <- "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
```

Comment:
we can update the uuid package and use the `uuid::UUIDValidate` function instead of hardcoding the pattern.

Reply:
There's even an `is_uuid` function [already implemented in the worker](https://github.com/hms-dbmi-cellenics/worker/blob/11b3d90da687a51a9206aeb4216d12c513196159/r/R/utilities.R#L271), you could use that instead. Needs a slight refactor.